dozer-ingestion/oracle/src/connector/mod.rs

```diff
@@ -524,7 +524,7 @@ mod tests {
         let data_user = "DOZER";
         let host = "database-1.cxtwfj9nkwtu.ap-southeast-1.rds.amazonaws.com";
         let sid = "ORCL";
-        let pid = None;
+        let pid = sid;
```

Comment:
This seems unrelated

Reply:
Yes,but the lint error shows it, i have undone this in the check2 commit though

Reply:
here the linter shows the same error ,should i ignore it?
https://github.com/getdozer/dozer/actions/runs/7812319344/job/21309138408?pr=2379